### DISCUSSION TITLE
Software vulnerability detection on monitored end-points? (future plans)

### DISCUSSION THREAD

**Discussion by lacak-sk** [utmstack/UTMStack#652] [Q&A] [answered]
Are you thinking about implementing the software vulnerability detection function on endpoints with an installed agent?
i.e. that UTMStack could identify unpatched software with known vulnerabilities on end-points (monitored by Windows/Linux/macOS agents).

Maybe for a start it would be enough if unpatched OSs were identified. As unpatched OSes are a security issue that should be identified ...

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @lacak-sk, we already have it, a Vulnerability Scanner as Stand-alone product, it doesn't detect unpatched software, at the beginning can detect a lot of vulnerabilities registered across the internet, will be available soon.
Best Regards.

**Reply by lacak-sk** [reply]
It's a nice feature I like in Wazuh (https://documentation.wazuh.com/4.7/user-manual/capabilities/vulnerability-detection/index.html) and which I miss in UTMStack.

**Reply by TYrorare** [reply]
I saw Vulnerabilities module in integrations "UTMSTACK nodules" in v9. But in v10 its disappear. And no AD audit. Mayby they are inside appliance?

**Reply by TYrorare** [reply]
And else... wazuh was in v9 but developers remove it. And no nids. that is bad. NIDS help to andestend what is in yuor network wrong
